packages/projects/src/components/ProjectForm.jsx

```diff
@@ -376,6 +417,23 @@ const ProjectForm = ({ mode, projectHandle }) => {
               fieldStatus={value(state, 'urlStatus')} value={value(state, 'url')}
             />
           </label>
+          <label className='block'>
+            <span className='text-gray-700'>Team</span>
+            <br />
+            <span className='text-gray-500 text-sm'>
+              Yourself or one of the groups you are in.
+            </span>
+            <select
+              className={formClass} value={state.owner}
+              onChange={changeSelectedOwner.bind(null, dispatch)}
+            >
+              {state.ownerOptions.map(ownerOption => (
```

Comment:
This throws an error initially because it doesn't like the empty array.

```
`value` prop on `select` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.
```

Reply:
I believe it's about the `value={state.owner}` being `null`, not the empty array, so I'll fix that.